.github/workflows/development.yml

```diff
@@ -39,6 +39,14 @@ jobs:
         ruby-version: ${{matrix.ruby}}
         bundler-cache: true
     
+    - name: Install liburing
+      run: |
+        git clone https://github.com/axboe/liburing.git
+        cd liburing
+        ./configure
+        make
+        sudo make install
```

Comment:
I think this step breaks non-ubuntu builds. I tried adding `if: ${{ matrix.ruby == '3.1' && matrix.os == 'ubuntu' }}` but this didn't work.

I'll fix this if we decide to proceed with this PR.